python/lsst/ts/m2gui/controltab/tab_config_view.py

```diff
@@ -33,7 +37,319 @@ class TabConfigView(TabDefault):
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        group = QGroupBox("In-Position Flag")
+
+        layout = QFormLayout()
+        layout.addRow("Axial Threshold: ", self._config_parameters["in_position_axial"])
+        layout.addRow(
+            "Tangent Threshold: ", self._config_parameters["in_position_tangent"]
+        )
+        layout.addRow("Sample Window: ", self._config_parameters["in_position_sample"])
+
+        group.setLayout(layout)
+
+        return group
+
+    def _create_group_communication_timeout(self):
+        """Create the group of communication timeout.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        group = QGroupBox("Communication Timeout")
+
+        layout = QFormLayout()
+        layout.addRow(
+            "SAL Communication Timeout: ", self._config_parameters["timeout_sal"]
+        )
+        layout.addRow(
+            "cRIO Communication Timeout: ", self._config_parameters["timeout_crio"]
+        )
+        layout.addRow(
+            "ILC Telemetry Receipt Error Persistance: ",
+            self._config_parameters["timeout_ilc"],
+        )
+
+        group.setLayout(layout)
+
+        return group
+
+    def _create_group_misc_inclinometer(self):
+        """Create the group of miscellaneous of inclinometer.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        group = QGroupBox("Miscellaneous (Inclinometer)")
+
+        layout = QFormLayout()
+        layout.addRow("Delta Range: ", self._config_parameters["misc_range_angle"])
+        layout.addRow(
+            "Difference Enabled: ", self._config_parameters["misc_diff_enabled"]
+        )
+
+        group.setLayout(layout)
+
+        return group
+
+    def _create_group_misc_temperature(self):
+        """Create the group of miscellaneous of temperature.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        group = QGroupBox("Miscellaneous (Temperature)")
+
+        layout = QFormLayout()
+        layout.addRow(
+            "Cell Temperature Delta: ",
+            self._config_parameters["misc_range_temperature"],
+        )
+
+        group.setLayout(layout)
+
+        return group
+
+    def _set_signal_config(self, signal_config):
+        """Set the config signal with callback function.
+
+        Parameters
+        ----------
+        signal_config : `SignalConfig`
+            Signal of the configuration.
+        """
+
+        signal_config.config.connect(self._callback_signal_config)
+
+    @Slot()
+    def _callback_signal_config(self, config):
+        """Callback of the config signal for the new configuration.
+
+        Parameters
+        ----------
+        config : `Config`
+            New configuration.
+        """
+
```

Comment:
This shall be replaced with:
```
for key,label in self._config_parameters.items():
    entry.setText(getattr(config, key))
```

If you need to show units, see what's available in cRIOpy - either create classes which will add units (probably better) or have a class which will take unit as arguments.

Reply:
As comment above, I don't think it is a good idea to have a complex `dict` structure. Thanks!